packages/s/sof-firmware/package.yml

```diff
@@ -1,19 +1,20 @@
 name       : sof-firmware
 homepage   : https://github.com/thesofproject/sof-bin
-version    : 2.7.1
-release    : 13
+version    : 2.7.2
+release    : 14
 source     :
-    - https://github.com/thesofproject/sof-bin/releases/download/v2023.09.1/sof-bin-2023.09.1.tar.gz : 583233652a298cb754c03f47c33a58c9b97770498aa0db03141f11ef705d191d
+    - https://github.com/thesofproject/sof-bin/releases/download/v2023.09.2/sof-bin-2023.09.2.tar.gz : 23063a3e447497bbb2683d0c5f3a0fbb248dabfb24544138be0e73e9e15e0f63
 license    :
     - BSD-3-Clause
     - ISC
 component  : kernel
 summary    : Sound Open Firmware
 description: |
     Sound Open Firmware
-install    : |
+install    : |-
```

Comment:
There is an extra `-`.

Reply:
I think that was added by go-task. Removed it. Will keep an eye on that in the future and file an issue if go-task is being silly.